src/controllers/statisticsController.js

```diff
@@ -0,0 +1,104 @@
+import models from '../db/models';
+
+const { Statistics } = models;
+
+/**
+ * @description class responsible for statistics of author's article readCount
+ *
+ * @class StatisticsController
+ */
+class StatisticsController {
+  /**
+   * @description - This method is responsible for counting updating article readCount
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} author - Username of the author of the article
+   *
+   * @returns {object} - object representing response message
```

Comment:
But no response message was returned from this method.

Reply:
It was an oversight. It has been updated.